functionalTest/.gitignore

```diff
@@ -0,0 +1,22 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+package-lock.json
```

Comment:
We need to keep package-lock.json - it shouldn't be ignored

Reply:
Removed package-lock.json from git ignore file.